src/acme_common.ml

```diff
@@ -8,7 +8,7 @@ let sha256_and_base64 a = Primitives.sha256 a |> B64u.urlencode
 
 let ( let* ) = Result.bind
 
-module J = Yojson.Basic
+module J = Yojson.Safe
```

Comment:
This is the other big change, moving from `Basic` to `Safe` since that's what is "supposed" to be used and also what `Jose` uses.